src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -4,87 +4,95 @@ short-title: Integrate Flutter
 description: Learn how to integrate a Flutter module into your existing iOS project.
 ---
 
-TODO(jmagman): add the pre-built framework alternative and review.
+Flutter can be embedded into your existing iOS application in a piecemeal fashion as
+frameworks or libraries.
 
-## Integration steps
+## Create a Flutter module
 
-To integrate a Flutter module into your app, use the following steps:
+In order to embed Flutter into your existing application, first create a Flutter module.
 
- 1. [Create Flutter Module](#create-a-flutter-module)
-   f Creates the Flutter project that is hosted in the existing app.
- 1. [Add Module Dependency in Existing
-    App](#make-the-host-app-depend-on-the-flutter-module)
-    Sets up Xcode to include your Flutter project into the build of your
-    existing app.
+From the command line, run:
 
-## Create a Flutter module
+```terminal
+cd some/path/
+flutter create --template module my_flutter
```

Comment:
maybe just -t?

Reply:
I was copying https://github.com/flutter/website/blob/master/src/docs/development/packages-and-plugins/developing-packages.md#developing-dart-packages-dart, I would expect verbose to be better for docs but I'll defer to @sfshaza2

Reply:
SG